API/Common.pm

```diff
@@ -107,13 +117,24 @@ sub _precacheAlbum {
 		$album->{genre} = $album->{genre}->{name};
 		$album->{image} = __PACKAGE__->getImageFromImagesHash($album->{image}) || '';
 
+		# If the user pref is for classical music enhancements to the display, is this a classical release or has the user added the genre to their custom classical list?
+		$isClassique = 0;
+		if ( $prefs->get('useClassicalEnhancements') ) {
+			if ( $album->{genres_list} && grep(/Classique/,@{$album->{genres_list}}) || $genreList{$album->{genre}} ) {
```

Comment:
Operator precedence can be confusing (at least it is for me). You have `&&` and `||` in the same condition. Please use braces to make sure the condition is evaluated correctly (and my poor brain understands the precedence of the expression).

Reply:
Done